.eslintrc.js

```diff
@@ -14,9 +14,11 @@ module.exports = {
     project: true,
     tsconfigRootDir: __dirname
   },
-
   rules: {
     eqeqeq: ['error', 'always'],
+    '@typescript-eslint/consistent-type-definitions': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/strict-boolean-expressions': 'off',
```

Comment:
actually i just saw this in hyun's pr and let's keep this rule intact - explicit null and explicit undefined checks are generally good things anyway

Reply:
sounds good!